synopsys-task/src/synopsys-task/model/blackduck.ts

```diff
@@ -0,0 +1,65 @@
+export enum BLACKDUCK_SCAN_FAILURE_SEVERITIES {
+  ALL = "ALL",
+  NONE = "NONE",
+  BLOCKER = "BLOCKER",
+  CRITICAL = "CRITICAL",
+  MAJOR = "MAJOR",
+  MINOR = "MINOR",
+  OK = "OK",
+  TRIVIAL = "TRIVIAL",
+  UNSPECIFIED = "UNSPECIFIED",
+}
+
+export interface Blackduck {
+  blackduck: BlackduckData;
+  github?: GithubData;
+}
+
+export interface BlackduckData {
+  url: string;
+  token: string;
+  install?: { directory: string };
+  scan?: {
+    full?: boolean;
+    failure?: { severities: BLACKDUCK_SCAN_FAILURE_SEVERITIES[] };
+  };
+  automation: AutomationData;
+}
+
+export interface Branch {
```

Comment:
Please remove the data, which is not required for Blackduck scan. Please look at line numbers 29-65